### ISSUE TITLE
Setting accessibilityRequired to false sets required prop to true

### ISSUE THREAD

**Issue by flip-it** [necolas/react-native-web#2451]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

`<TextInput accessibilityRequired={false}  />` correctly sets the dom attribute `aria-required="false"` but it also sets `required`. This makes the element required even though it’s intended not to.

[Documentation](https://necolas.github.io/react-native-web/docs/accessibility/#accessibility-props-api) states `accessibilityRequired: ?boolean` which means false should be an allowed value.

### Expected behavior

`required` to not be set when `accessibilityRequired={false}`

### Steps to reproduce

1. Insert a `<TextInput accessibilityRequired={false}  />`.
2. Inspect the DOM and look for `required` attribute.

### Test case

https://codesandbox.io/embed/green-http-gpil9z

**Comment by necolas** [comment]
Thanks, this is a bug on this line that just sets the value to `true` rather than the boolean provided https://github.com/necolas/react-native-web/blob/1aa84d54479fb64e77c86d064263144370195f86/packages/react-native-web/src/modules/createDOMProps/index.js#L241
